common/schemes/wc_stasis_trap_scheme.txt

```diff
@@ -0,0 +1,248 @@
+﻿stasis_trap_scheme = {
+	skill = intrigue
+	hostile = yes
+
+	power_per_skill_point = 2.0
+	resistance_per_skill_point = 1.75
+	power_per_agent_skill_point = 1
+	
+	minimum_progress_chance = 10
+	maximum_progress_chance = 90
+	minimum_success = 5
+	maximum_secrecy = 85
+	maximum_success = 85
+	
+	allow = {
+		has_perk = stasis_trap # Scheme unlocked by the Stasis Trap
+		
+		is_adult = yes
+		is_landed = yes
+		is_imprisoned = no
```

Comment:
So I would created basic triggers like `allow_schemes_trigger` including adult and imprisoned checks.